src/test/java/com/eleks/academy/whoami/core/state/SuggestingCharacterTest.java

```diff
@@ -102,4 +153,73 @@ void moveCharactersTest() {
 		}
 	}
 
+	@Test
+	void nickNameNotChangeAfterShufflingTest() {
```

Comment:
you can check this in the previous test as well, no need to copy all the test just to check another field

Reply:
Resolved
Good remark.